packages/devtools_app/lib/src/timeline/html_timeline_screen.dart

```diff
@@ -40,7 +40,7 @@ import 'timeline_protocol.dart';
 
 // TODO(kenz): connect a frame's UI and GPU code in the full_timeline.
 
-const enableMultiModeTimeline = false;
+const enableMultiModeTimeline = true;
```

Comment:
you intended to turn this on by default with this CL, right?
you might want to hide this behind a flag so that just customers interested in the feature can try it before you make it the default? I'm also fine with making it the default if this was intentional.